### PR TITLE
Unit test for bytes flow

### PR DESCRIPTION
**Pull request by g4s8**
#39 - added unit tests for text and json implementations of bytes flow.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#40](https://github.com/artipie/docker-adapter/issues/40) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/a7dfcdba-e39c-4f37-8304-715b92f8a7bf); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/docker/misc/BytesFlowAsTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    Flowable.fromArray(
+                        new ByteArray(
+                            "{\"one\":1}".getBytes(StandardCharsets.UTF_8)
+                        ).boxedBytes()
+                    )
+                )
+            ).future().get(),
+            Matchers.equalTo(
+                Json.createObjectBuilder()
+                    .add("one", 1)
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/docker/misc/BytesFlowAsTest.java]: @g4s8 what if we factor this out into its own variable and use the `toString()` representation as the input to `ByteArray`?

File: src/test/java/com/artipie/docker/misc/BytesFlowAsTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ * Test case for {@link BytesFlowAs}.
+ *
+ * @since 1.0
+ */
+final class BytesFlowAsTest {
+
+    @Test
+    void parsesText() throws Exception {
+        final String txt = "hello";
+        final Charset charset = StandardCharsets.UTF_8;
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/docker/misc/BytesFlowAsTest.java]: @g4s8 It's a constant, we can inline it in both places.

**Comment by g4s8** [comment]
@carlosmiranda thanks, should be fixed now, see https://github.com/artipie/docker-adapter/pull/40/commits/3ff358b746049f2f221d6dd40383c9664732d26d

**Comment by 0crat** [comment]
Job was finished in 2 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**